samples/bluetooth/peripheral/prj.conf

```diff
@@ -11,6 +11,8 @@ CONFIG_BT_ATT_PREPARE_COUNT=5
 CONFIG_BT_BAS=y
 CONFIG_BT_HRS=y
 CONFIG_BT_IAS=y
+CONFIG_BT_CTS=y
+CONFIG_BT_CTS_HELPER_API_ENABLE=y
```

Comment:
Reference to old name here as well - seems like you didn't test the latest change? :)

Reply:
My bad, I have retest latest push.
Thanks!